product_docs/docs/pem/8/pem_online_help/06_toc_pem_bart_management/index.mdx

```diff
@@ -11,7 +11,7 @@ Postgres Enterprise Manager (PEM) is designed to assist database administrators,
 
 The EDB Backup and Recovery Tool (BART) is an administrative utility providing simplified backup and recovery management for multiple local or remote EDB Postgres Advanced Server and PostgreSQL database servers. For more information about BART, please visit the EnterpriseDB website at:
```

Comment:
```suggestion
The EDB Backup and Recovery Tool (BART) is an administrative utility providing simplified backup and recovery management for multiple local or remote EDB Postgres Advanced Server and PostgreSQL database servers. See the [Backup and Recovery](<https://www.enterprisedb.com/enterprise-postgres/edb-postgres-backup-and-recovery-tool>) page on the EDB web site for more information about BART.
```

Reply:
The page we are referring them to doesn't have much info and has a link back to the docs. It might be better to just send them to the docs here.